lib/src/database/database.dart

```diff
@@ -1,29 +1,57 @@
+library vscout_cli.database;
+
 import 'dart:io';
 
 import 'package:sembast/sembast.dart';
 import 'package:sembast/sembast_io.dart';
 import 'package:path/path.dart';
 import 'package:uuid/uuid.dart';
 
-String dbPath = '/../database/vscout.db';
-var pathToDb = (dirname(Platform.script.toFilePath()).toString() + dbPath);
-
 class DatabaseHandler {
   Database db;
   Store store;
+  File databaseFile;
+  String relativeDatabasePath;
+  String realDatabasePath;
+  Map resultFields;
 
-  DatabaseHandler() {
-    // Constructors can't call async functions, proper initialization is done in [InitializeDb()].
+  static final DatabaseHandler _singleton = new DatabaseHandler._internal();
+
+  factory DatabaseHandler() {
+    return _singleton;
+    // Constructors can't call async functions, actual initialization is done in [InitializeDb()].
+  }
+
+  DatabaseHandler._internal() {
+    //
   }
 
   InitializeDb() async {
-    this.db = await this.newDb();
+    this.resultFields = new Map();
+    this.resultFields['status'] = null;
```

Comment:
https://dart.dev/guides/language/effective-dart/usage#dont-use-an-explicit-default-value-of-null